### ISSUE TITLE
FID on CIFAR-100 at 100k steps is much higher than expected (≈90) when rerunning the provided training script

### ISSUE THREAD

**Issue by thanhlamauto** [submission-neurips22/few-shot-diffusion-models#14]
Hi maintainers, thanks a lot for open-sourcing this project — really appreciate the work.

I'm trying to reproduce the CIFAR-100 results using the exact command provided in the repo, but I'm seeing a much higher FID than expected.

### What I did
I ran the training script exactly as written:

uv run bash -x script/main.sh 0 vfsddpm_cifar100_vit_film_mean_sigma

Observed behavior

At 100k steps, the FID is ~90 on my rerun.  This seems significantly worse than what I expected from the reported/claimed results.I also attached samples and a training/eval screenshot. I expected to see a substantially lower FID by 100k steps (or at least close to the repo’s reported behavior).

Could you help confirm whether my evaluation setup matches yours? In particular:

- Which checkpoint is used for FID evaluation? (EMA vs non-EMA)

- How many samples are generated for FID, and with what batch size?

- Are you using torch-fidelity / pytorch-fid / clean-fid, and which exact settings?

- Are the Inception statistics computed with the same pipeline (CIFAR-100 train set stats vs precomputed stats)?

- Any required flags/configs for CIFAR-100 (e.g., image range/normalization, resize, center-crop) that could strongly affect FID?

<img width="274" height="274" alt="Image" src="https://github.com/user-attachments/assets/e833b84a-8303-4c83-99b2-7aa7a4a65d8d" />

<img width="2985" height="2387" alt="Image" src="https://github.com/user-attachments/assets/89006a01-103a-4031-ae82-3890fa48e13d" />
